src/TableRecordInstantiator.php

```diff
@@ -38,12 +38,6 @@ public function instantiate(Fixture $fixture)
         $tablename = substr($fixture->getClass(), 6);
         $r = new TableRecord($tablename);
 
-        if (!isset($this->ids[$tablename])) {
```

Comment:
@igormukhingmailcom what if my tables need automatic ids? is there option to enable that?